pyscriptjs/examples/index.html

```diff
@@ -36,6 +36,9 @@ <h2 class="text-2xl font-bold text-blue-600"><a href="./todo.html" target=”_bl
     <h2 class="text-2xl font-bold text-blue-600"><a href="./todo-pylist.html" target=”_blank”>PyScript Native TODO App</a></h2>
     <p>Demo showing how would a Simple TODO App would look like only using PyScript native elements</code> tag</p>
 
+    <h2 class="text-2xl font-bold text-blue-600"><a href="./fetch-data.html" target=”_blank”>PyScript Fetch API</a></h2>
+    <p>Demo showing how to use fetch API client.</code> tag</p>
```

Comment:
The demo looks really nice and I think the description doesn't give it honor. Maybe something like:

`Bitcoin Rate Dashboard showing how to use fetch API client.`

?